agent/util-scripts/pbench-tool-meister-stop

```diff
@@ -42,6 +38,88 @@ def is_running(pid):
     return True
 
 
+def wait_for_pid(pid):
+    """wait_for_pid - wait for a process to actually stop running.
+    """
+    while is_running(pid):
+        time.sleep(0.1)
```

Comment:
See https://pubs.opengroup.org/onlinepubs/9699919799/functions/wait.html.

Reply:
Could you be more specific?  All I could find in that reference was a recommendation that portable applications _use_ `waitpid()` (i.e., rather than `wait()`).